lib/parse-args.js

```diff
@@ -191,18 +188,13 @@ function buildYargs (withCommands = false) {
 function hideInstrumenterArgs (yargv) {
   let argv = process.argv.slice(1)
   argv = argv.slice(argv.indexOf(yargv._[0]))
-  if (argv[0][0] === '-') {
-    argv.unshift(process.execPath)
-  }
   return argv
 }
 
 function hideInstrumenteeArgs () {
   let argv = process.argv.slice(2)
   const yargv = parser(argv)
 
-  if (!yargv._.length) return argv
-
   // drop all the arguments after the bin being
   // instrumented by c8.
   argv = argv.slice(0, argv.indexOf(yargv._[0]))
```

Comment:
This looks like an early exit.  This condition was never reached per the former code coverage report.  Perhaps another test case?